pkg/raft/rafttest/interaction_env_handler_add_nodes.go

```diff
@@ -35,6 +35,7 @@ func (env *InteractionEnv) handleAddNodes(t *testing.T, d datadriven.TestData) e
 	n := firstAsInt(t, d)
 	var snap pb.Snapshot
 	cfg := raftConfigStub()
+	var asyncWrites bool
```

Comment:
The plan is still to remove this, alongside `AdvanceHack`, right?

- `de_fortification_basic`: I'm not sure why it's used here
- `async_storage_writes.txt`
- `async_storage_writes_append_aba_race.txt`

I looked at all the commits and everything looks good, but I think I'm missing the bigger picture.

The `Node.syncWrites` comment ends mid-sentence, btw.

Reply:
Yes, or make it less hacky because it's still convenient to have some kind of "synchronous writes" way in tests.

`de_fortification_basic` <- let's ask Arul
`async_storage_writes` <- what is the question about these ones?

Re bigger picture, I'm still coming up with it on the fly. I have good idea on how I want the API to look like, but there are a lot of tests on the way, written under a different assumption. Ideally, something ergonomic for tests too. Put some thoughts in slack.

> The Node.syncWrites comment ends mid-sentence, btw.

Will fix the comment.

Reply:
Fixed the comment, and renamed the option to `asyncWrites`, to better match what we used to have.